company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise8Test.java

```diff
@@ -149,7 +159,7 @@ public void whoOrderedSaucers()
     public void ordersByCustomerUsingAsMap()
     {
         MutableMap<String, MutableList<Order>> customerNameToOrders =
-                this.company.getCustomers().toMap(null, null);
+                this.company.getCustomers().toMap(Customer::getName, Customer::getOrders);
```

Comment:
Requesting the same formatting change here as well.

Reply:
Done